tests/pass/ptr_int_from_exposed.rs

```diff
@@ -2,6 +2,7 @@
 // Tree Borrows doesn't support int2ptr casts, but let's make sure we don't immediately crash either.
 //@[tree]compile-flags: -Zmiri-tree-borrows
 //@[stack]compile-flags: -Zmiri-permissive-provenance
+//@[tree]only-miri: miri specific warnings
```

Comment:
```suggestion
//@only-miri: miri specific warnings
```
Let's just disable these entirely, the other aliasing model might start warning any time.